### PR TITLE
Disallow passing headers object to fetcher

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

File: services/Instagram.history.json [hunk 1]
```diff
@@ -1,52 +1,37 @@
 {
   "Developer Terms": [
     {
-      "fetch": "https://developers.facebook.com/terms",
+      "fetch": "https://developers.facebook.com/terms?locale=en_GB",
       "select": "#faq-page",
-      "headers": {
```
Comment (martinratinaud) [inline services/Instagram.history.json:6]: This is wrong as you may know. The history should contain all the historical configuration.
But I'm fine with it if you are

**Review by martinratinaud — APPROVED** [review]
As discussed before, I still think we will have to have a functionality at some point where it's easy to bypass certain limitations that we may encounter.
passing headers or cookies will at some point be needed IMO.

File: services/Instagram.history.json [hunk 2]
```diff
@@ -1,52 +1,37 @@
 {
   "Developer Terms": [
     {
-      "fetch": "https://developers.facebook.com/terms",
+      "fetch": "https://developers.facebook.com/terms?locale=en_GB",
```
Comment (martinratinaud) [inline services/Instagram.history.json:4]: Nice!

**Review by martinratinaud — APPROVED** [review]

File: services/Instagram.history.json (hunk 1 above)
Comment (Ndpnt) [reply services/Instagram.history.json:6]: > As discussed before, I still think we will have to have a functionality at some point where it's easy to bypass certain limitations that we may encounter. passing headers or cookies will at some point be needed IMO.

I may agree with that, but currently we do not have the need, so for contribution and code simplicity we remove the feature for now.

**Comment by Ndpnt** [comment]
> As discussed before, I still think we will have to have a functionality at some point where it's easy to bypass certain limitations that we may encounter. passing headers or cookies will at some point be needed IMO.

I may agree with that, but currently we do not have the need, so for contribution and code simplicity we remove the feature for now.

**Pull request merged**